BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/EndpointUpdatedHandler.cs

```diff
@@ -11,26 +13,69 @@ namespace BookingQueueSubscriber.Services.MessageHandlers
     public class EndpointUpdatedHandler : IMessageHandler<EndpointUpdatedIntegrationEvent>
     {
         private readonly IVideoApiService _videoApiService;
+        private readonly ILogger<EndpointUpdatedHandler> _logger;
+        private const int RetryLimit = 3;
+        private const int RetrySleep = 3000;
 
-        public EndpointUpdatedHandler(IVideoApiService videoApiService)
+        public EndpointUpdatedHandler(IVideoApiService videoApiService, ILogger<EndpointUpdatedHandler> logger)
         {
             _videoApiService = videoApiService;
+            _logger = logger;
         }
 
         public async Task HandleAsync(EndpointUpdatedIntegrationEvent eventMessage)
         {
-            var conference = await _videoApiService.GetConferenceByHearingRefId(eventMessage.HearingId);
+            ConferenceDetailsResponse conference = await _videoApiService.GetConferenceByHearingRefId(eventMessage.HearingId);
```

Comment:
Can use var here instead of ConferenceDetailsResponse

Reply:
Updated object type of conference from ConferenceDetailsResponse to var